### PR TITLE
Fix rosidl_parser verbatim comments

### PR DESCRIPTION
**Pull request by pablogs9**
Solves https://github.com/ros2/rosidl/issues/676

### CODE REVIEW

**Comment by pablogs9** [comment]
CC: @sloretz

**Comment by pablogs9** [comment]
Friendly ping @sloretz @clalancette

File: rosidl_parser/rosidl_parser/definition.py [hunk 1]
```diff
@@ -448,7 +448,7 @@ def get_comment_lines(self):
         ]
         lines = []
         for comment in comments:
-            lines.extend(comment.splitlines())
+            lines.extend([x.strip() for x in comment.splitlines()])
```
Comment (sloretz) [inline rosidl_parser/rosidl_parser/definition.py:451]: Stripping the whitespace might be ok for comments, but it looks like there's a larger problem with parsing string literals. I've opened an alternative to this PR in #688.

**Comment by pablogs9** [comment]
Superseded by https://github.com/ros2/rosidl/pull/688

**Pull request closed without merging**